custom_components/keymaster/translations/en.json

```
@@ -64,5 +64,11 @@
         "title": "Confirm"
       }
     }
+  },
+  "system_health": {
+    "info": {
+      "zwave_integration": "Zwave Integration",
```

```suggestion
      "zwave_integration": "Z-Wave Integration",
```
this is for consistency with how Z-Wave is listed everywhere else in the frontend